src/views/ErrorHandler.vue

```diff
@@ -1,25 +1,52 @@
-<template></template>
+<template>
+    <div v-if="title" class="container pad-bottom">
+        <SmallPage>
+            <Loader
+                :title="title"
+                :state="state"
+                :message="message"
+                :mainAction="mainActionText"
+                :alternativeAction="alternativeActionText"
+                @main-action="mainActionHandler"
+                @alternative-action="alternativeActionHandler"
+                />
+        </SmallPage>
+    </div>
+</template>
```

Comment:
Probably need to remove the empty template from classes inheriting from `ErrorHandler` (`CheckoutErrorHandler`, `LoginErrorHandler`, `SignMessageErrorHandler`, `SignupErrorHandler`) as it probably overwrites this one.